datagrid_gtk3/utils/transformations.py

```diff
@@ -448,5 +449,9 @@ def degree_decimal_str_transform(value, length=8):
     if len(value) < length:
         value = '0' * (length - len(value)) + value
```

Comment:
I didn't notice in the first time, but isn't this if the same as `value.zfill(length)`?

Not related to the PR itself, just to mention =P